runners/flink/src/test/java/org/apache/beam/runners/flink/ReadSourcePortableTest.java

```diff
@@ -81,7 +99,8 @@ public static void tearDown() throws InterruptedException {
 
   @Test(timeout = 120_000)
   public void testExecution() throws Exception {
```

Comment:
Are we missing a test case for bounded read? Current validates runner test suites won't cover it.